lib/entities/organization.ts

```diff
@@ -33,9 +27,12 @@ import createOrganizationApi from '../create-organization-api'
  * @param  {Object} data - API response for a Organization
```

Comment:
Could you please delete `{ }` from all comments? 

```suggestion
 * @param data - API response for an Organization
```

Reply:
@suevalov sure! done! ✅